source/agora/common/Config.d

```diff
@@ -463,8 +469,8 @@ private Config parseConfigImpl (in CommandLine cmdln, Node root)
                     return get!(ushort, "interfaces", "port")(cmdln, &l);
                 return opt!(ushort, "interfaces", "port")(cmdln, &l, defaultPort);
             }();
-
-            interfaces ~= InterfaceConfig(type, address, port);
+            auto tls = opt!(bool, "interfaces", "tls")(cmdln, &l, false);
```

Comment:
You can just use `get`, `bool` fields are defined as always optional.

Reply:
It now uses enum `type` for https.